module/Activity/src/Activity/Service/Activity.php

```diff
@@ -170,15 +173,45 @@ public function createActivity(array $params, $dutch, $english)
             );
         }
 
-        assert($dutch || $english, "Activities should have either be in dutch or english");
-
         $em = $this->getServiceManager()->get('Doctrine\ORM\EntityManager');
 
         // Find the creator
+        /** @var \User\Model\User $user */
         $user = $em->merge(
             $this->getServiceManager()->get('user_role')
         );
 
+
+        // Find the organ the activity belongs to, and see if the user has permission to create an activity
+        // for this organ
+        $organId = intval($params['organ']);
+
+        $organ = null;
+
+        if ($organId === 0) {
+            /** @var \Decision\Service\Member $memberService */
+            $memberService = $this->getServiceManager()->get('decision_service_member');
+            $member = $memberService->findMemberByLidNr($user->getLidnr());
+
+            // The organs that the user belongs to with the correct organId (either 0 or 1)
+            $organs = $memberService->getOrgans($member);
+
+            // An array only containing the organ that this member belongs to and with the correct id
+            $correctOrgan = array_filter($organs, function (Organ $organ) {
+                return $organ->getId();
+            });
+
+            // Check if the member belongs to the organ
+            if (count($organ) === 0) {
```

Comment:
What does this do? Isn't $organ equal to null? I don't quite understand this whole block of code does actually. Are you trying to determine if the current user is in an organ with id $organId?